src/components/Table/GridTable.tsx

```diff
@@ -347,6 +349,9 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
       let isCard = false;
       if (matches) {
         isCard = nestedCards && nestedCards.maybeOpenCard(row);
+        if (isCard && nestedCards) {
```

Comment:
Is `nestedCards` a sort of feature flag?

Reply:
Yeah, it turns on all of the "extra DOM elements" work needed to do the card styling.